diff --git a/contracts/RewardsManager.sol b/contracts/RewardsManager.sol
index 7d413ee..fd15ae7 100644
--- a/contracts/RewardsManager.sol
+++ b/contracts/RewardsManager.sol
@@ -442,7 +442,7 @@ contract RewardsManager is ReentrancyGuard {
         } else if (to == address(0)) {
             _handleWithdrawal(vault, from, amount);
         } else {
-            _handleTransfer(vault, from, to, amount);
+            _handleTransfer(vault, to, from, amount);
         }
     }
 
